### ISSUE TITLE
Custom Background Feature

### ISSUE THREAD

**Issue by JamesRyan09** [AndBible/and-bible#2569]
I really like this app but the background makes me fell lack, for my suggestion it will be more interesting to read if we can add custom Background where we can add images from our gallery since I want to use a scroll theme and more relaxing to read.

Pls consider My suggestion, Thank you!

-JamesRyan09
-jamesryan.torremocha@gmail.com

**Comment by tuomas2** [comment]
This is possible to be done with [Custom CSS](https://github.com/AndBible/and-bible/wiki/Custom-CSS-for-modules-for-And-Bible). If you implement such a module and it looks good, we can make a add-on module out of it and release it in AndBible repository.

**Comment by JamesRyan09** [comment]
Yes pls add such thing, but it is better if the user can add directly from
their internal storage or photo gallery it is just an option but if not
possible just do as you said a module background, I am happy for
considering my suggestion, the developers of this app are attentive God
bless you all.
